hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/InodeTree.java

```diff
@@ -81,6 +88,19 @@ enum ResultKind {
   private List<RegexMountPoint<T>> regexMountPointList =
       new ArrayList<RegexMountPoint<T>>();
 
+  private final boolean isNestedMountPointSupported;
+  private Set<LinkEntry> sortedLinkEntries = new TreeSet<>((o1, o2) -> {
```

Comment:
its a little weird that `sortedLinkEntries` is a private member, but only used in `getLinkEntries` which just does an `addAll` and return. I think we should probably get rid of this as a member.
otherwise we need to consider clearing the set. It also just seems like an unnecessary memory overhead.

Reply:
I agree we can get rid of a a member. I originally used it as a member but later return in a function and didn't remove it.